UKHomeOffice/cerberus-service
# COP-11361: Add test for Add capability to remove and add items from collections

## Description
Add test for capability to remove or add items from collections
## To Test
npm run cypress:test:local -- --spec cypress/integration/airpax/issue-aipax-tasks.spec.js

Should verify capability to Add or Remove items from collections

Should submit a target successfully from a AirPax task

## Developer Checklist

\* Required
